src/metatrain/utils/data/target_info.py

```diff
@@ -166,7 +166,10 @@ def _check_layout(self, layout: TensorMap) -> None:
                 )
 
         if self.is_spherical:
-            if layout.keys.names != ["o3_lambda", "o3_sigma"]:
+            if len(layout.keys.names) < 2 or layout.keys.names[:2] != [
+                "o3_lambda",
+                "o3_sigma",
+            ]:
```

Comment:
@frostedoyster do you think this will break something? Essentially the concatenated features of MACE have irreps that look like:

```python
o3.Irreps("2x0e + 2x1o + 2x0e")
```

(i.e. they have two scalar parts that are separated). I could reorder them so that I have `"4x0e + 2x1o"` but the most natural thing I believe is to keep their order. To convert those features into a `TensorMap` I need that blocks have some extra key that can differentiate the first scalar block from the last one, so I have keys `["o3_lambda", "o3_sigma", "i_irrep"]`.

Reply:
Can properties be non-unique? I.e. `i_irrep=[0,0,0,1,1]`?